PROJECT

```diff
@@ -36,7 +33,4 @@ resources:
   kind: SelfNodeRemediationConfig
   path: github.com/medik8s/self-node-remediation/api/v1alpha1
   version: v1alpha1
-  webhooks:
```

Comment:
the config still has a webhook...

Reply:
Nope, it's marked in red to show it's removed.

Reply:
let me rephrase: we still have a webhook for the SelfNodeRemediationConfig CRD, and this is its webhook config, which should not be deleted :)